### PR TITLE
Updated styles on ManageCase page and updated Axioswithauth on Render…

### PR DESCRIPTION
**Pull request by katieolson84**
## Description

Updated Axios calls on RenderHome to be Axios with auth. Updated the styling on ManageCases. Styling will need to be adjusted again once more data is displayed to ensure data is displayed without errors.

Moved the logo in the Header to the styles folder for global access. 

Converted .css to .less for ManageCases 

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by nwestallen — APPROVED** [review]
Styling looks much nicer now and I don't notice any issues

File: src/components/pages/Home/RenderHomePage.js [hunk 1]
```diff
@@ -85,8 +85,8 @@ function RenderHomePage(props) {
   const user = useContext(UserContext);
 
   useEffect(() => {
-    axios
-      .get(`${process.env.REACT_APP_API_URI}/cases`, {
+    axiosWithAuth()
+      .get(`/cases`, {
         headers: {
```
Comment (WeiWu2) [inline src/components/pages/Home/RenderHomePage.js:90]: axiosWithAuth takes care of headers and authorization so these should be removed

Comment (katieolson84) [reply src/components/pages/Home/RenderHomePage.js:90]: Good catch! I will go in and remove the headers and push a new commit.

**Review by WeiWu2 — CHANGES_REQUESTED** [review]
headers should be removed from all the axioWithAuth calls since it already takes care of it, everything else looks fine

**Review by JonathanMary — APPROVED** [review]

**Comment by JonathanMary** [comment]
Good to go.

**Review by WeiWu2 — APPROVED** [review]

**Pull request merged**